descriptions/SDK.trueSKY.md

```diff
@@ -0,0 +1 @@
+[**trueSKY**](https://simul.co/) is the leading cloud, atmosphere and weather tool kit.
```

Comment:
Let's avoid saying things like `leading`. And a bit more descriptive explanation would be great.

Reply:
i just copy-pasted their info, but i'll try to find something else